codespell_lib/_codespell.py

```diff
@@ -26,7 +26,7 @@
 import re
 import sys
 
-word_regex_def = u"[\\w\\-'’`]+"
+word_regex_def = u"[a-zA-Z0-9\\-'’`]+"
```

Comment:
What about the `é` of `attaché` or `fiancée`?

Reply:
I guess that is why we'd need the test that ensures the default word regex matches everything in the dictionaries